From 8c847d62998556288d6160d645cdad96ea309bed Mon Sep 17 00:00:00 2001
From: DogEZ <127666673+DdogezD@users.noreply.github.com>
Date: Sun, 5 Oct 2025 05:33:19 +0800
Subject: [PATCH] disable all io sched to fallback "none"

"none" is showing better performance on our device. it meaning to let ufs to handle it by itself
however, oneplus set it to "bfq", so we need to do this to make a fallback
---
 arch/arm64/configs/gki_defconfig | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index dbfa1c1addb4..cdbeac5c94ee 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -110,8 +110,10 @@ CONFIG_BLK_CGROUP_IOCOST=y
 CONFIG_BLK_CGROUP_IOPRIO=y
 CONFIG_BLK_INLINE_ENCRYPTION=y
 CONFIG_BLK_INLINE_ENCRYPTION_FALLBACK=y
-CONFIG_IOSCHED_BFQ=y
-CONFIG_BFQ_GROUP_IOSCHED=y
+# CONFIG_MQ_IOSCHED_DEADLINE is not set
+# CONFIG_MQ_IOSCHED_KYBER is not set
+# CONFIG_IOSCHED_BFQ is not set
+# CONFIG_BFQ_GROUP_IOSCHED is not set
 CONFIG_GKI_HACKS_TO_FIX=y
 # CONFIG_CORE_DUMP_DEFAULT_ELF_HEADERS is not set
 CONFIG_BINFMT_MISC=y
